iroh/src/commands/sync.rs

```diff
@@ -354,14 +354,14 @@ impl AuthorCommands {
 /// Format the content. If an error occurs it's returned in a formatted, friendly way.
 async fn fmt_content(doc: &Doc, entry: &Entry, mode: DisplayContentMode) -> Result<String, String> {
     let read_failed = |err: anyhow::Error| format!("<failed to get content: {err}>");
-    let decode_failed = |_err: std::string::FromUtf8Error| "<invalid UTF-8>".to_string();
+    let encode_hex = |err: std::string::FromUtf8Error| format!("0x{}", hex::encode(err.as_bytes()));
```

Comment:
Not really a showstopper, but how do you distinguish between the actual text `0x1234` and the values 0x1234 formatted like this?

To make the representation unique so that you can do a roundtrip, you would have to add quotes to the text format.

Reply:
this is the reason why I asked if you should be able to explicitly say give me hex, but ultimately if your content is hex encoded and added as a value, it's valid utf8 and I doubt the distinction matters

Reply:
Another alternative would be to `utf8: texthere` and `hex: encodedhere`might be clearer